app/assets/images/payment_icons/touchngo.svg

```diff
@@ -0,0 +1 @@
/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><defs><path id="pi-SVGID_1_" d="M10.4 3.4h17.2v17.2H10.4z"/></defs><clipPath id="pi-SVGID_2_"><use href="#pi-SVGID_1_" overflow="visible"/></clipPath><g clip-path="url(#pi-SVGID_2_)"><defs><path id="pi-SVGID_3_" d="M10.4 3.4h17.2v17.2H10.4z"/></defs><clipPath id="pi-SVGID_4_"><use href="#pi-SVGID_3_" overflow="visible"/></clipPath><path class="st3" d="M23.7 3.4h-9.4c-2.2 0-3.9 1.8-3.9 3.9v9.4c0 2.2 1.8 3.9 3.9 3.9h9.4c2.2 0 3.9-1.8 3.9-3.9V7.3c0-2.2-1.7-3.9-3.9-3.9"/><path d="M23.7 4.1h-9.4c-1.8 0-3.2 1.4-3.2 3.2v9.4c0 1.8 1.4 3.2 3.2 3.2h9.4c1.8 0 3.2-1.4 3.2-3.2V7.3c0-1.8-1.4-3.2-3.2-3.2" clip-path="url(#pi-SVGID_4_)" fill="#295daa"/><path class="st3" d="M17.4 10.6h-.8c-.2 0-.6.2-.6.4l.1-.4h-1l-.4 2.5h1l.3-1.6h1.1s-.1.8-.2 1.1c-.1.4.2.4.3.4h.5l.3-1.9c-.2-.2-.4-.5-.6-.5M14.2 10.6l-.2 1.1.8-1.1zM15.4 6.7h-3.3l-.2.8h1.2l-.5 2.7h1l.5-2.7h1.2z"/><path class="st3" d="M16.1 9.4l.1-.8h-.9l-.1.8h.9m-1.4.8c-.3 0-.5-.2-.4-.5l.3-1.5c0-.3.3-.5.6-.5h1.7c.3 0 .5.2.4.5L17 9.7c0 .3-.3.5-.6.5h-1.7zM21.3 9.4l.1-.8h1.3l.1-.8h-1.7c-.3 0-.5.2-.6.5l-.3 1.5c0 .3.1.5.4.5h1.6l.1-.8h-1zM19.3 9.4h-1.1l.3-1.7h-.9l-.3 2c0 .3.1.5.4.5h.9c.3 0 .5-.2.7-.4 0 0 .1.3.2.4.1.1.6 0 .6 0l.4-2.5h-.9l-.3 1.7zM25.6 7.7h-.8c-.2 0-.5.2-.6.4l.2-1.3h-.9l-.5 3.4h.9l.3-1.6h1s-.1.8-.2 1.1c-.1.4.2.4.3.4h.4l.3-1.9c.1-.3-.1-.5-.4-.5M22.3 13.9c-.4 0-.6-.3-.6-.7l.3-2c.1-.4.4-.7.8-.7H25c.4 0 .6.3.6.7l-.3 2c-.1.4-.4.7-.8.7h-2.2m2-.9l.3-1.5h-1.5l-.3 1.5h1.5zM20.3 13h-1.5l.3-1.5h2.4l.2-.9H19c-.4 0-.7.3-.8.7l-.3 2c-.1.4.2.7.6.7h2.1c.4 0 .7-.3.8-.7l.2-1.2h-1l-.3.9z"/><path class="st5" d="M18.2 15.1L17 17.7h-.8l-.1-1.6-.8 1.6h-.8l-.1-2.6h.5l.1 2.1 1-2.1h.6l.1 2.1 1-2.1zM20.6 17.7h-.5l.5-2.6h.6zM21.6 17.7H21l.5-2.6h.6zM25.4 17.7H25c-.2 0-.4 0-.4-.1-.1 0-.1-.1-.1-.2v-.2l.4-2.1h.6l-.1.6h.5l-.1.5h-.5l-.3.8v.1c0 .1 0 .1.1.1h.4l-.1.5z"/><path class="st5" d="M18.2 15.1L17 17.7h-.8l-.1-1.6-.8 1.6h-.8l-.1-2.6h.5l.1 2.1 1-2.1h.6l.1 2.1 1-2.1zM20.6 17.7h-.5l.5-2.6h.6zM21.6 17.7H21l.5-2.6h.6zM25.4 17.7H25c-.2 0-.4 0-.4-.1-.1 0-.1-.1-.1-.2v-.2l.4-2.1h.6l-.1.6h.5l-.1.5h-.5l-.3.8v.1c0 .1 0 .1.1.1h.4l-.1.5z"/><path class="st5" d="M18.2 15.1L17 17.7h-.8l-.1-1.6-.8 1.6h-.8l-.1-2.6h.5l.1 2.1 1-2.1h.6l.1 2.1 1-2.1zM20.6 17.7h-.5l.5-2.6h.6zM21.6 17.7H21l.5-2.6h.6zM25.4 17.7H25c-.2 0-.4 0-.4-.1-.1 0-.1-.1-.1-.2v-.2l.4-2.1h.6l-.1.6h.5l-.1.5h-.5l-.3.8v.1c0 .1 0 .1.1.1h.4l-.1.5zM14 15.8c-.1 0-.2-.1-.4-.1h-.8c-.2 0-.4 0-.5.1-.2.2-.3.3-.3.5l-.2.8V17.4s0 .1.1.1h1.9l.1-.5h-1.3l-.1-.1v-.1H14l.1-.7v-.2l-.1-.1m-.5.7h-1v-.1s0-.1.1-.1H13.3c.1 0 .2 0 .2.1v.1M20 15.9s-.1-.1 0 0c-.1-.1-.2-.1-.3-.1H18v.4h1.3s.1 0 .1.1v.1h-1.1c-.1 0-.2 0-.2.1-.1 0-.1.1-.1.2s-.1.2-.1.3v.5l.1.1h1.9l.3-1.4V16c-.2 0-.2 0-.2-.1m-1.6 1.3c-.1 0-.1 0-.1-.1V17l.1-.1h.8l-.1.3h-.7zM24.2 15.8c-.1 0-.2-.1-.4-.1H23c-.2 0-.4 0-.5.1-.1.1-.2.2-.2.4l-.3 1V17.5s0 .1.1.1H24l.1-.5h-1.3l-.1-.1v-.1h1.5l.1-.7V16c0-.1 0-.1-.1-.2m-.5.7h-1v-.1s0-.1.1-.1H23.5c.1 0 .2 0 .2.1v.1"/></g></svg>
```

Comment:
Can we remove all unused `class` and `id` such as `class="st3"` and `pi-SVGID_1_`?

Reply:
Hi @tauthomas01 ,
Sorry forgot to include boost icon so I have added it in latest [commit](https://github.com/activemerchant/payment_icons/pull/393/commits/30611929eeb39ff2a3ed062ecf259a013549b8b8) now.
Regarding the attributes in touchngo icon. I tried removing them but it causes icon colors to change. They are not the unused attributes as removing them affects the icon.